lib/puppet/module/task.rb

```diff
@@ -45,7 +45,7 @@ def initialize(task_name, module_name)
     end
 
     FORBIDDEN_EXTENSIONS = %w{.conf .md}
-    MOUNTS = %w[lib files tasks]
+    MOUNTS = %w[files lib scripts tasks]
```

Comment:
Could you add a unit test for this? It looks like [`PuppetSpec::Modules.create`](https://github.com/puppetlabs/puppet/blob/a4be41ff96769e784ec1f6825350b7a3c09478cc/spec/lib/puppet_spec/modules.rb#L24-L49) needs updating to accept `scripts` option, and then update [`spec/unit/module_spec.rb`](https://github.com/puppetlabs/puppet/blob/a4be41ff96769e784ec1f6825350b7a3c09478cc/spec/unit/module_spec.rb#L562-L568) to verify `Puppet::Module#task_file` returns files in the `scripts` directory? Might be good to add the `lib` and `files` cases too unless those are tested elsewhere?

Reply:
Totally!